pom.xml

```diff
@@ -247,6 +247,14 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.jacoco</groupId>
+      <artifactId>org.jacoco.agent</artifactId>
+      <classifier>runtime</classifier>
+      <version>0.7.5.201505241946</version>
```

Comment:
since this version number is duplicated in multiple locations, IMO would be better to extract it into property and this also should simplify testing of updates